apps/codeforafrica/src/lib/api.ghost/posts.js

```diff
@@ -0,0 +1,109 @@
+import { promises as fs } from "fs";
+import path from "path";
+
+import GhostContentAPI from "@tryghost/content-api";
+import camelcaseKeys from "camelcase-keys";
+
+import equalsIgnoreCase from "@/codeforafrica/utils/equalsIgnoreCase";
+
+const cacheDir = path.join(process.env.PWD, "public/data");
+
+function initializeContentAPI() {
+  return new GhostContentAPI({
+    url: process.env.GHOST_URL,
+    key: process.env.GHOST_API_KEY,
+    version: process.env.GHOST_API_VERSION || "v5.0",
+  });
+}
+
+function transformPost(post) {
+  const {
+    customExcerpt,
+    excerpt: originalExcerpt,
+    primaryTag,
+    publishedAt: publishedAtRaw,
+    slug,
+    tags: originalTags,
+    ...other
+  } = camelcaseKeys(post, { deep: true });
+
+  const excerpt = customExcerpt || originalExcerpt || null;
+  const href = `/${primaryTag.slug}/${slug}`;
+  const publishedAt = new Date(publishedAtRaw).toLocaleDateString("en", {
+    year: "numeric",
+    month: "short",
+    day: "numeric",
+  });
+  const tags = originalTags
+    .filter(
+      (t) =>
+        !(
+          equalsIgnoreCase(t.name, "opportunities") ||
+          equalsIgnoreCase(t.name, "stories")
+        )
+    )
```

Comment:
Having thought about this further, I think we can simplify it to just:
```suggestion
      (t) => !equalsIgnoreCase(t.name, primaryTag.name)
    )
```

`primaryTag.name` will always be `Opportunities` or `Stories`, right?